src/kademlia/dht_storage.cpp

```diff
@@ -346,6 +345,7 @@ namespace
 		}
 
 		void put_immutable_item(sha1_hash const& target
+			, sha1_hash const& our_id
```

Comment:
I have a couple question I can't fully answer myself.
- One `get_immutable_item` request can come from any DHT, but what's the probability that a `put_immutable_item` with the same hash comes from multiples DHT having distinct IDs?
- Why `put_mutable_item` does not follow the same rule to discard "the least important one"?